### DISCUSSION TITLE
[Help]: Slides glitches after I resize the windows while autoscroll and loop are on

### DISCUSSION THREAD

**Discussion by Irjean** [davidjerleke/embla-carousel#742] [Help] [answered]
### Summary

Hello !

I'm trying to build an infinite scrolling carousel, but after resizing the window the carousel will struggle to follow the slides placements at the end of the count. Not sure if it's a bug or if there's a problem in my settings.

Any help is greatly appreciated 🙏 

https://github.com/davidjerleke/embla-carousel/assets/66118881/ef2440ba-f928-4d33-ab5c-c56414e90914

### If applicable, which variants of Embla Carousel are relevant to this question?

- [ ] embla-carousel **(Core)**
- [X] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [x] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### CodeSandbox example

https://codesandbox.io/p/sandbox/embla-carousel-drag-free-react-forked-x3svqx

**Comment by davidjerleke** [comment]
@Irjean thanks! I'll investigate this when possible 👍.

**Reply by davidjerleke** [reply]
@Irjean can you reproduce the problem in [this CodeSandbox](https://codesandbox.io/p/sandbox/embla-carousel-drag-free-react-forked-9h396f)?

Best,
David

**Comment by davidjerleke** [comment, marked as answer]
- @Irjean this will be tracked in #750 from now on. Closing this.

**Reply by Irjean** [reply]
Sure, thank you !

**Reply by davidjerleke** [reply]
@Irjean thanks for reporting the bug 👍🏻.
